Backend/Middleware-APD/src/main/java/middlewareapd/model/JWToken.java

```diff
@@ -0,0 +1,37 @@
+package middlewareapd.model; // Ensure this matches your directory structure
+
+import java.time.LocalDateTime;
+import lombok.Data;
+
+/**
+ * Represents a JWT (JSON Web Token) with associated metadata.
+ * This class contains information about the token, user session details, and access times.
+ */
+@Data
+public class JWToken {
+    private String jwt;               // The JWT string
+    private String uuid;              // Unique identifier for the user
+    private int isAdmin;             // User's admin status (0 for false, 1 for true)
+    private LocalDateTime lastLogin;  // Timestamp of the last login
+    private LocalDateTime logout;     // Timestamp of the logout
+    private LocalDateTime lastAccess; // Timestamp of the last access
```

Comment:
i think for the `lastAccessed` field could define like "the last time the token was used in a request" might be clearer as to what it is